src/components/Dashboard/Dashboard.test.js

```diff
@@ -1,9 +1,83 @@
 import React from 'react';
-import { Dashboard } from './Dashboard';
-import { mount} from 'enzyme';
+import { mount } from 'enzyme';
+import { Provider } from 'react-redux';
+import configureMockStore from 'redux-mock-store'
+import thunk from 'redux-thunk';
+import { BrowserRouter as Router} from 'react-router-dom';
+import Dashboard from './Dashboard';
 
-describe('Header', () => {
-  it.skip('renders', () => {
-    mount(<Header />);
+const mockStore = configureMockStore([thunk]);
```

Comment:
This process with thunks looks complex. Thank you for taking initiative on this.